packages/opentelemetry-cloud-trace-exporter/src/types.ts

```diff
@@ -14,6 +14,8 @@
 
 import {Metadata} from '@grpc/grpc-js';
 
+// extends cloudtrace_v2.Schema$Span
```

Comment:
Why is this a comment instead of actually extending that interface?

Reply:
Oops I just forgot to delete that. I was trying exactly that but the generated types are not very useful (`string | null`) so I'd rather copy in an enum. It is `string | null` because JS `number` can't represent large integers, but that is no concern here. They get serialized just fine from the enum